### DISCUSSION TITLE
Setting up Flood to work on the local network

### DISCUSSION THREAD

**Discussion by needlesslygrim** [jesec/flood#603] [Q&A]
I have set up Flood and it works great, I can access it at ```127.0.0.1:3000```. However if I try to access it outside of localhost, e.g. on my phone connected to the same network, then I get a unable to connect error in Firefox. How to I enable Flood to only be accessible on the local network? Thanks in advance.

**Comment by clomads** [comment]
i have mine running via terminal and this is what made it work for me

`flood --host "0.0.0.0"`
